tests/containers/ReportContainer_spec.js

```diff
@@ -611,4 +611,22 @@ describe('Report Container', () => {
       done();
     }, 5001);
   });
+
+  it('Generate test empty table template', () => {
```

Comment:
we should have test for all charts ,

we can modify the TemplateProvider.getTestLayoutTemplate(); 
using prepareSections function, 
and to remove all the data from the template